projects/packages/classic-theme-helper/src/custom-content-types.php

```diff
@@ -34,18 +35,74 @@ function jetpack_load_custom_post_types() {
 	add_action( 'jetpack_activate_module_custom-content-types', array( '\Automattic\Jetpack\Classic_Theme_Helper\Jetpack_Testimonial', 'activation_post_type_support' ) );
 
 	add_action( 'init', array( '\Automattic\Jetpack\Classic_Theme_Helper\Nova_Restaurant', 'init' ) );
+
+	$site_id = Connection_Manager::get_site_id();
+	if ( is_wp_error( $site_id ) ) {
+		return;
+	}
+	add_action( 'rest_api_init', 'register_rest_route_custom_content_types' );
+
 }
 
 if ( ! function_exists( 'jetpack_custom_post_types_loaded' ) ) {
 	/**
-	 * Make module configurable.
+	 * Pass the active status to the front-end in it's initial state.
 	 */
 	function jetpack_custom_post_types_loaded() {
-		if ( class_exists( 'Jetpack' ) ) {
-			Jetpack::enable_module_configurable( __FILE__ );
-		}
+		$initial_state = 'var CUSTOM_CONTENT_TYPE__INITIAL_STATE; typeof CUSTOM_CONTENT_TYPE__INITIAL_STATE === "object" || (CUSTOM_CONTENT_TYPE__INITIAL_STATE = JSON.parse(decodeURIComponent("' . rawurlencode(
```

Comment:
This variable follows the same style as a couple of other initial states, such as `JP_CONNECTION_INITIAL_STATE`: https://github.com/Automattic/jetpack/blob/9a9f08e7bf866d9bc5c8b090191acabb54edfb92/projects/packages/connection/src/class-initial-state.php#L60